x-pack/plugins/maps/public/classes/sources/es_search_source/load_index_settings.js

```diff
@@ -29,14 +29,14 @@ async function fetchIndexSettings(indexPatternTitle) {
   const http = getHttp();
   const toasts = getToasts();
   try {
-    const indexSettings = await http.fetch(`../${INDEX_SETTINGS_API_PATH}`, {
+    const indexSettingsPath = `/${INDEX_SETTINGS_API_PATH}`;
+    return await http.fetch(indexSettingsPath, {
```

Comment:
remove `indexSettingsPath` variable and just put `/${INDEX_SETTINGS_API_PATH}` in fetch call since its not used any where else